gax-java/gax-grpc/src/test/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProviderTest.java

```diff
@@ -284,14 +284,6 @@ public void testDirectPathXdsDisableByDefault() throws IOException {
     assertThat(provider.isDirectPathXdsEnabled()).isFalse();
   }
 
-  @Test
```

Comment:
This is a new test added 4 days. Are we suppose to remove it?

Reply:
Not sure why this was removed (possibly some mistake when merging). Adding it back.